myapp/db/migrate/20230302002602_change_tasks_description_string_to_text_of.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeTasksDescriptionStringToTextOf < ActiveRecord::Migration[7.0]
+  def up
+    change_table :tasks do |t|
+      t.change :description, :text
```

Comment:
Change description column data type to text from string so that user can enter long sentences for descriptions